README.md

```diff
@@ -17,35 +17,35 @@ Click on the name of a plugin or module to view that content's documentation:
 ### Roles
 Name | Description
 --- | ---
-[cloud.azure_roles.load_balancer](https://github.com/ansible-collections/cloud.azure_roles/blob/main/roles/load_balancer/README.md)|A role to manage Azure Load Balancer.
-[cloud.azure_roles.managed_postgresql](https://github.com/ansible-collections/cloud.azure_roles/blob/main/roles/managed_postgresql/README.md)|A role to manage Azure PostGreSQL Database.
-[cloud.azure_roles.network_interface](https://github.com/ansible-collections/cloud.azure_roles/blob/main/roles/network_interface/README.md)|A role to manage Azure Network Interface.
-[cloud.azure_roles.networking_stack](https://github.com/ansible-collections/cloud.azure_roles/blob/main/roles/networking_stack/README.md)|A role to manage Azure Networking Stack.
-[cloud.azure_roles.resource_group](https://github.com/ansible-collections/cloud.azure_roles/blob/main/roles/resource_group/README.md)|A role to manage Azure Resource Group.
-[cloud.azure_roles.security_group](https://github.com/ansible-collections/cloud.azure_roles/blob/main/roles/security_group/README.md)|A role to manage Azure Security Group.
-[cloud.azure_roles.virtual_machine](https://github.com/ansible-collections/cloud.azure_roles/blob/main/roles/virtual_machine/README.md)|A role to manage Azure Virtual Machine.
+[cloud.azure_ops.azure_load_balancer_with_public_ip_with_public_ip](https://github.com/ansible-collections/cloud.azure_ops/blob/main/roles/azure_load_balancer/README.md)|A role to manage Azure Load Balancer.
+[cloud.azure_ops.azure_managed_postgresql](https://github.com/ansible-collections/cloud.azure_ops/blob/main/roles/azure_rds_postgresql/README.md)|A role to manage Azure PostGreSQL Database.
+[cloud.azure_ops.azure_network_interface](https://github.com/ansible-collections/cloud.azure_ops/blob/main/roles/azure_network_interface/README.md)|A role to manage Azure Network Interface.
+[cloud.azure_ops.azure_networking_stack](https://github.com/ansible-collections/cloud.azure_ops/blob/main/roles/azure_networking_stack/README.md)|A role to manage Azure Networking Stack.
+[cloud.azure_ops.azure_resource_group](https://github.com/ansible-collections/cloud.azure_ops/blob/main/roles/azure_resource_group/README.md)|A role to manage Azure Resource Group.
+[cloud.azure_ops.azure_security_group](https://github.com/ansible-collections/cloud.azure_ops/blob/main/roles/azure_security_group/README.md)|A role to manage Azure Security Group.
```

Comment:
I think for most of these they could be `azure_manage_whatever`.  For example, the README of the security group role starts with `A role to manage an Azure Security Group`.  So a descriptive name would be similar to that, something like `cloud.azure_ops.manage_security_group`.

Reply:
updated